### PR TITLE
Enable drilldown resizing when there's one drilldown item

### PR DESCRIPTION
**Pull request by djsauble**
The problem is that if a drilldown has only a detail panel and no masters, it won’t resize.

I’ve fixed this problem by having the Drilldown controller reference the detail panel when a master isn’t available. For now, the only feature which requires this is the Loggers feature.

### CODE REVIEW

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Drilldown.js [hunk 1]
```diff
@@ -393,6 +393,7 @@ Ext.define('NX.controller.Drilldown', {
    * Given N drilldown items, this panel should have a width of N times the current screen width
    */
   syncSizeToOwner: function () {
+    console.log("syncSizeToOwner()");
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Drilldown.js]: cruft

**Comment by kellyrob99** [comment]
Minimally I think we need to document why this is required for this edge-case, otherwise it inevitably becomes copy-pasta for the future.

**Comment by jdillon** [comment]
why would a drilldown view not have masters?

**Comment by djsauble** [comment]
Documentation added, cancel button fixed.

Drilldown provides a bunch of default behavior for masters. Because the list of loggers is such a snowflake (it’s simultaneously a master and a detail view), it can’t use any of that inherited code. Thus, I’ve made it a detail view instead of a master.

**Comment by kellyrob99** [comment]
+1

**Comment by mcculls** [comment]
+1

**Pull request closed without merging**